contrib/gcppubsub/samples/README.md

```diff
@@ -12,112 +20,110 @@
    gcloud services enable pubsub.googleapis.com
    ```
 
-1. Setup
-   [Knative Eventing](https://github.com/knative/docs/tree/master/eventing).
 1. Install the
-   [in-memory `ClusterChannelProvisioner`](https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel).
-   - Note that you can skip this if you choose to use a different type of
-     `Channel`. If so, you will need to modify `channel.yaml` before deploying
-     it.
-1. Create a `Channel`. You can use your own `Channel` or use the provided
-   sample, which creates `qux-1`. If you use your own `Channel` with a different
-   name, then you will need to alter other commands later.
+   [GCP PubSub Source from this yaml](../config/default-gcppubsub.yaml) from
+   source:
 
    ```shell
-   kubectl -n default apply -f samples/gcp-pubsub-source/channel.yaml
+   ko apply --filename https://github.com/knative/eventing-sources/tree/master/contrib/gcppubsub/config/default-gcppubsub.yaml
```

Comment:
My preference is to use local paths (relative to repo root), so that these instructions can be used during development. Then the docs repo would use the released versions.

And what does it mean to `ko apply` the yaml from master? Does that mean any local changes to the binary would be honored, but not from the yaml?

Reply:
Maybe this sample should go in favor of the public docs? (I ended up copying about 1/3rd of the public docs in here, because they have a much clearer flow and more repeatable command-line flow for getting the GCP side set up.)

Reply:
It's not clear whether these docs are development instructions (at which point, they are probably too detailed) or usage instructions that we'd point end-users at. Right now they are awkwardly between the two.

Reply:
I think we will move the instructions from knative/docs if and when we move this source to the GoogleCloudPlatform org. At that point, these instructions will merge with the knative/docs version.

I almost deleted this sample, but I thought that it might be worth exploring how small the documentation could be in terms of objects, and I think this does simplify the footprint in terms of the documentation. I removed the link to Knative serving here now, too.